fileio/private/ft_senstype.m

```diff
@@ -247,6 +247,10 @@
 elseif issubfield(input, 'orig.FileHeader') &&  issubfield(input, 'orig.VarHeader')
   % this is a complete header that was read from a Plexon *.nex file using read_plexon_nex
   type = 'plexon';
+
+elseif issubfield(input, 'orig.FileHeader') &&  issubfield(input, 'orig.VarHeaders')
```

Comment:
I notice that the difference between the 'plexon' and 'nex5' senstype is just one single character in one of the fieldnames of the original header structure. This is as intended?

Reply:
I did not notice ft_senstype.m code at first and simply like VarHeaders as field name more than VarHeader. And then it was handy in ft_senstype. I can check 
orig.FileHeader.NexFileHeader == 'NEX5'
instead, for example.

Reply:
Ah, I see. Well I don't know who came up with the original VarHeader (so without the 's') but if this is based on a convention that is also used in other software tools, I wonder whether it's clever to adopt a slightly different fieldname for something that probably has the same content. Therefore, I would recommend sticking to the 'without-s' naming convention (even if you don't like it :) ) and do a more principled check for the 'NEX5' string.